src/stars.coffee

```diff
@@ -60,8 +61,22 @@ class Stars extends Command
   installPackages: (packages, callback) ->
     return callback() if packages.length is 0
 
-    commandArgs = packages.map ({name}) -> name
-    new Install().run({commandArgs, callback})
+    toName = ({name}) -> name
+
+
+    list = new List()
+    listOptions =
+      argv:
+        dev: true
+        bare: true
+
+    list.listUserPackages listOptions, (error, userPackages) ->
+      userPackages = userPackages.map(toName)
+
+      commandArgs = packages.map(toName).filter (name) ->
+        return name not in userPackages
+
+      new Install().run({commandArgs, callback})
```

Comment:
How can I get a reference to `Install` or `Stars` from the spec? I'm thinking repeat the steps for the ` "when the install flag is specified" -> it "installs all of the stars"`, then run the same command a second time and verify that the list is empty by the time it's passed over to `new Install().run(...)`